app/services/katello/pulp3/repository/ansible_collection.rb

```diff
@@ -16,6 +16,10 @@ def remote_options
           end
         end
 
+        def create_options
+          common_create_options
+        end
```

Comment:
this can be moved to app/services/katello/pulp3/repository.rb and then the yum part can override it

Reply:
and this prevents the need for every single class to define it

Reply:
Thanks. I have revised the section again.